hedwig.py

```diff
@@ -0,0 +1,344 @@
+from bibtexparser import loads
+import os.path
+import sys
+import time
+import yaml
+
+from auth_match import Auth_Match
+from owlery import Connection
+import queries
+import wos
+from spiderowl import WOSnnection
+
+ARTICLE_COUNT = 0
+JOURNAL_COUNT = 0
+PUBLISHER_COUNT = 0
+AUTHOR_COUNT = 0
+
+def get_config(config_path):
+    try:
+        with open(config_path, 'r') as config_file:
+            config = yaml.load(config_file.read())
+    except Exception, e:
+        print("Error: Check config file")
+        print(e)
+        exit()
+    return config
+
+def bib2csv(bib_data):
+    csv_data = {}
+    row = 0
+    col_names = set(y for x in bib_data.entries for y in x.keys())
+
+    for x in bib_data.entries:
+        row += 1
+        csv_data[row] = {}
+
+        for col_name in col_names:
+            v = x.get(col_name, '')
+            v = v.replace('\n', ' ')
+            v = v.replace('\r', ' ')
+            v = v.replace('\t', ' ')
+            csv_data[row][col_name] = v.encode('utf-8').strip()
+    return csv_data
+
+def process(connection, data):
+    global ARTICLE_COUNT
```

Comment:
A lot is happening in this function and there is global here. Could this be broken up? Is the global necessary?